MotorControl/MotorControl/Resources/Finger_Tapping.json

```diff
@@ -43,7 +43,8 @@
                 "identifier":"fingerTappingInstructions",
                 "type":"instruction",
                 "title":"You will finger tap for 30 seconds with each hand",
-                "text":"Speed and accuracy of finger tapping can show us the fine motor control of your hands. Your doctor might measure this differently.",
+                "text":"This activity measures your tapping speed for each hand. Your medical provider may measure this differently.",
+                "fullInstructionsOnly": true,
```

Comment:
Figma says 

mPower: "Speed of finger tapping can reflect severity of motor symptoms in Parkinson’s disease. This activity measures your tapping speed for each hand. Your medical provider may measure this differently."

pkYou: "Speed of finger tapping can show us whether you have trouble with movement or action in your hands. Your PKU doctor might measure this differently."